src/data/Dhis2TrackedEntityInstances.ts

```diff
@@ -441,44 +441,54 @@ async function getTeisFromApi(options: {
     enrollmentStartDate?: Moment;
     enrollmentEndDate?: Moment;
     ouMode: RelationshipOrgUnitFilter;
-}): Promise<PaginatedTeiGetResponse> {
+}): Promise<TrackedEntitiesResponse> {
     const { api, program, orgUnits, page, pageSize, enrollmentStartDate, enrollmentEndDate, ouMode } = options;
 
     const fields: TeiKey[] = [
-        "trackedEntityInstance",
+        "trackedEntity",
         "inactive",
         "orgUnit",
         "attributes",
         "enrollments",
         "relationships",
-        "featureType",
         "geometry",
     ];
 
     const ouModeQuery =
         ouMode === "SELECTED" || ouMode === "CHILDREN" || ouMode === "DESCENDANTS"
-            ? { ouMode, ou: orgUnits?.map(({ id }) => id) }
-            : { ouMode };
+            ? { orgUnitMode: ouMode, orgUnit: orgUnits?.map(({ id }) => id) }
+            : { orgUnitMode: ouMode };
+
+    const filters: TrackedEntityGetRequest = {
+        ...ouModeQuery,
+        program: program.id,
+        pageSize: pageSize,
+        page: page,
+        totalPages: true as const,
+        fields: fields.join(","),
+        enrollmentEnrolledAfter: enrollmentStartDate?.format("YYYY-MM-DD[T]HH:mm"),
+        enrollmentEnrolledBefore: enrollmentEndDate?.format("YYYY-MM-DD[T]HH:mm"),
```

Comment:
In the old implementation the order parameter was present: `order: "created:asc"` but is not being passed in the new implementation. If it's no necessary anymore that's ok just checking in case we're missing the value.